SIC #34 summary (#39)
stateless-consensus/eth-stateless-book: src/development/sic-calls/history.md

[@GabRocheleau](https://x.com/GabRocheleau) for @EFJavaScript: I made progress on the tree conversion code and plan to start running existing execution-spec-tests to check the correctness of the implementation.

[@gballet](https://x.com/gballet) from geth/[@StatelessEth](https://x.com/StatelessEth): continued working on merging tree conversion logic into geth master. He also published a [Holesky shadowfork](https://stateless.fyi/development/advanced/holesky-shadowfork.html) page in the stateless book, which explains how to easily run geth in devnet mode using Holesky state.

[@jasoriatanishq](https://x.com/jasoriatanishq) for Nethermind: has been wrapping the tree conversion implementation, now passing all tests. He is also working on rebasing the working branch on Pectra.

### 2. EIP-7612: follow up on code conversion when touching an account

The discussion focused on EIP-7612's behavior in specific scenarios, namely:

> - When a storage slot of an address is written to.

Reviewer: ```suggestion
- When the storage slot of a contract is written to.
```
